CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Smithy Gradle Plugin Changelog
 
+## 0.5.3 (TBD)
```

Comment:
This should also mention moving to Gradle to 6.8.3. I could also add a separate PR for that.

Reply:
Yeah a separate PR is fine